magisk/service.sh

```diff
@@ -37,6 +37,9 @@ fi
     # avoid breaking Realme fingerprint scanners
     resetprop ro.boot.flash.locked 1
 
+    # avoid breaking Oppo fingerprint scanners
```

Comment:
I suggest to update the comment, since it works for OnePlus as well (confirmed on OP7Pro A11, the same issue on banking apps gets resolved with this).

```suggestion
    # avoid breaking Oppo/OnePlus fingerprint scanners
```

Reply:
Nah, I intentionally put Oppo because OOS 12 is ColorOS.

Reply:
> I suggest to update the comment, since it works for OnePlus as well (confirmed to work on OP7Pro A11 Stock ROM, the same issue on banking apps gets resolved with this).

And realme. The three of them can be collectively referred to as OPlus lmao.